assets/js/base/hocs/with-reviews.js

```diff
@@ -106,10 +106,14 @@ const withReviews = ( OriginalComponent ) => {
 				offset: reviewsToSkip,
 			};
 
-			if ( categoryIds && categoryIds.length ) {
-				args.category_id = Array.isArray( categoryIds )
-					? categoryIds.join( ',' )
-					: categoryIds;
+			const categories = Array.isArray( categoryIds )
+				? categoryIds
+				: JSON.parse( categoryIds );
+
+			if ( categories && categories.length ) {
+				args.category_id = Array.isArray( categories )
```

Comment:
If we are checking `categories.length` above, are we not already confident at this point that it is an array?

Reply:
Unless `categories` can be a string at this point, which I think it can?

Reply:
Hmm, in theory, it should always be an array after line `111`.
But, if for some reason the `JSON.parse( categoryIds )` would return a string, we would have an issue.

Reply:
I've refactored it a bit, we need to check that it has a value, otherwise, it breaks.